references/d3-demo/src/trigger/python/agent.py

```
@@ -0,0 +1,171 @@
+        headers = {
+            "Authorization": f"Bearer {wait_token.publicAccessToken}",
+            "Content-Type": "application/json"
+        }
+        response = requests.post(url, json={"data": result}, headers=headers)
+        response.raise_for_status()
+        return response.json()
+
+basic_info_agent = Agent(
+    name="Basic Info Agent",
+    instructions=f"""You are an expert at extracting basic information from a person's contact details.
+    """,
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)